commons/pom.xml

```diff
@@ -75,6 +75,17 @@
       <artifactId>quarkus-junit5</artifactId>
       <scope>test</scope>
     </dependency>
+
+    <!--    Quarkus Wiremock extensions-->
```

Comment:
Nit: This comment should be omitted. The dependency's name already says what it is, and you already grouped them such that it makes it more obvious that both are related.